### ISSUE TITLE
island rank placeholder not working

### ISSUE THREAD

**Issue by kqrmo** [rlf/uSkyBlock#1253]
_Please paste the output from `/usb version` below_
```
Name: uSkyBlock
Version: 2.8.7
Description: Ultimate SkyBlock v2.8.7-73e1fc4
Language: en (en_GB)
State: d=128, r=64, i=2, p=2, n=true, awe=false
Server: Paper git-Paper-143 (MC: 1.15.2)
State: online=true, bungee=true
------------------------------
Vault 1.7.3-b${env.TRAVIS_BUILD_NUMBER} (ENABLED)
WorldEdit 7.1.0;8e55131 (ENABLED)
WorldGuard 7.0.2-SNAPSHOT;bf2593e (ENABLED)
Multiverse-Core 4.1.1-SNAPSHOT-b777 (ENABLED)
------------------------------
```

_What steps will reproduce the problem?_
1. /papi parse krmm_ %uskyblock_island_rank%

_If you have any log-files, please paste them to [pastebin.com](http://pastebin.com)_
* server-log: https://pastebin.com/XAtUdahH

(Not following the template will result in closing the issue.)

**Comment by dorianxonic** [comment]
@kqrmo how did you manage to make it work on 1.15.2?

**Comment by Muspah** [comment]
Out of curiosity, which plugin are you using to test this with? I'd like to replicate your environment to do some testing :)

**Comment by kqrmo** [comment]
> Out of curiosity, which plugin are you using to test this with? I'd like to replicate your environment to do some testing :)
Uhm.. You mean chat plugin? CMI 

> @kqrmo how did you manage to make it work on 1.15.2?

They've updated it to 1.15. Search uSkyblock Revived (?)

**Comment by Muspah** [comment]
Yeah, I'm not that known with placeholder usage, we only use some of Maxim's plugins (Featherboard et cetera). Thanks for the info.

I'm the publisher on Spigot: https://www.spigotmc.org/resources/uskyblock-revived.66795/

**Comment by dorianxonic** [comment]
@Muspah I'll open a separate issue
@kqrmo I'm usign 2.8.7

**Comment by Muspah** [comment]
Okay, thank you!
